corehq/apps/reports/standard/cases/basic.py

```diff
@@ -148,7 +148,7 @@ class CaseListReport(CaseListMixin, ProjectInspectionReport, ReportDataSource):
     def get_subpages(cls):
         def _get_case_name(request=None, **context):
             if 'case' in context:
-                return mark_safe(context['case'].name)
+                return mark_safe(context['case'].name) or _('View Case')
```

Comment:
nit: you could alternatively do:
```python
if 'case' in context and context['case'].name:
```

Reply:
I do like that better, don't like repeating user-facing text.